Source/WebCore/loader/ResourceMonitor.cpp

```diff
@@ -155,6 +154,12 @@ ResourceMonitor* ResourceMonitor::parentResourceMonitorIfExists() const
     return document ? document->parentResourceMonitorIfExists() : nullptr;
 }
 
+ResourceMonitor* ResourceMonitor::eligibleParentResourceMonitorIfExists() const
+{
+    auto* resourceMonitor = parentResourceMonitorIfExists();
```

Comment:
This could be changed to `RefPtr resourceMonitor = parentResourceMonitorIfExists()`, but I don't think it's required, since `isEligible()` is a trivial method.